client/src/pages/About.tsx

```diff
@@ -6,22 +6,55 @@ function About(this: any) {
     <div>
       <NavigationBar />
       <hr className="h-px mt-2 border-0 bg-gray-300"></hr>
-      <p
-        id="AboutDesc"
-        className="flex items-center items-baseline justify-start space-x-4 text-xs ml-60"
-      >
-        Text Insights is your go-to solution for harnessing the power of Natural
-        Language Processing (NLP). Our product empowers users to effortlessly
-        summarise and analyse text content, providing invaluable insights.
-        Whether you're a student grappling with voluminous textbooks, an
-        educator seeking efficient content evaluation, or a professional
-        navigating the digital landscape, Text Insights has you covered. We
-        prioritise simplicity, customisation, and accuracy, making complex tasks
-        a breeze. Powered by open-source models, our tool offers an ethical and
-        transparent approach to text processing. With Text Insights, you unlock
-        a world of concise understanding and sentiment analysis, enriching your
-        text-related endeavours.
-      </p>
+      <div className="mx-auto max-w-2xl p-4 mt-10 text-center"> 
+        <h1 className="text-3xl font-extrabold">About</h1> 
+        <p className="mt-4 text-sm font-bold">
```

Comment:
Can we add the `id="AboutDesc"` back, please? The Cypress test is failing because it has been removed.

Reply:
I just did. Thanks Grace.